guardian/editorial-collaboration
# Add local postgres database to setup

## What does this change?

Adds a local Postgres database in a Docker container to facilitate local development.

## How to test

Run `./scripts/setup`.

Does the script run without errors?

Can you see a new editorial-collaboration Docker container with a Postgres instance?
